docs/design.html

```diff
@@ -385,8 +385,10 @@ <h3 class="anchor-heading"><a id="replication" class="anchor-link"></a><a href="
     have the option of either waiting for the message to be committed or not, depending on their preference for tradeoff between latency and durability. This preference is controlled by the acks setting that the
     producer uses.
     Note that topics have a setting for the "minimum number" of in-sync replicas that is checked when the producer requests acknowledgment that a message
-    has been written to the full set of in-sync replicas. If a less stringent acknowledgement is requested by the producer, then the message can be committed, and consumed,
-    even if the number of in-sync replicas is lower than the minimum (e.g. it can be as low as just the leader).
+    has been written to the full set of in-sync replicas. If a less stringent acknowledgment is requested by the producer, then the message can be committed earlier and easier.
```

Comment:
"earlier and easier" is pretty vague, suggest more specific like: 
```suggestion
    has been written to the full set of in-sync replicas. If a less stringent acknowledgment is requested by the producer, then the message is committed asynchronously across the set of in-sync replicas if <code>acks=0</code>, or synchronously only on the leader if <code>acks=1</code>.
```
and if we make this change then change `In both cases` on the next line to `Regardless of the <code>acks</code> setting`